build-tools/packages/build-cli/docs/vnext.md

```diff
@@ -0,0 +1,32 @@
+`flub vnext`
+============
+
+Vnext commands are new implementations of standard flub commands using new infrastructure.
+
+* [`flub vnext check latestVersions VERSION RELEASE_GROUP`](#flub-vnext-check-latestversions-version-release_group)
+
+## `flub vnext check latestVersions VERSION RELEASE_GROUP`
+
+Determines if an input version matches a latest minor release version. Intended to be used in the Fluid Framework CI pipeline only.
+
+```
+USAGE
+  $ flub vnext check latestVersions VERSION RELEASE_GROUP [-v | --quiet]
```

Comment:
If we're going to have lots of commands with multiple args (like this one) as opposed to flags, then I think we need to define some conventions about the order, since args are order-dependent.

I suggest that the release group always come first, since it seems like the most likely arg that all/most commands would use.

If this ends up being unwieldy or confusing, we can always add a flag that overrides the argument if present, etc.